app/models/stripe_dispute.rb

```diff
@@ -38,7 +38,7 @@ def serialize_on_update(input)
       object_json = self.object
     end
 
-    self.balance_transactions = JSON.generate(object_json['balance_transactions'])
+    self.balance_transactions = object_json['balance_transactions']
```

Comment:
jsonb based columns are handled differently in Rails 5. Now they're loaded as an object, not the string.